### DISCUSSION TITLE
Next/image with unknown hostname

### DISCUSSION THREAD

**Discussion by fakehungry** [vercel/next.js#49563] [Help]
### Summary

I'm trying to figure out how to deal with unknown hostname in next/image. I try this [approach](https://stackoverflow.com/questions/66949606/what-is-the-best-way-to-have-a-fallback-image-in-nextjs), but it didn't work. Is there anyway to solve this issue.

**Comment by icyJoseph** [comment]
I think you need to use remote patterns: https://nextjs.org/docs/pages/api-reference/components/image#remotepatterns

Do you know, the path name at least?

**Comment by fakehungry** [comment]
In my case, I already setup all domains in next.config.js. However, I'd like to deal with upcoming unconfig hostname.
